cico_setup.sh

```diff
@@ -55,19 +55,34 @@ function run_tests_without_coverage() {
   make docker-test-unit-no-coverage
   make integration-test-env-prepare
   trap cleanup_env EXIT
+
+  # Check that postgresql container is healthy
+  check_postgres_healthiness
+
   make docker-test-migration
   make docker-test-integration-no-coverage
   make docker-test-remote-no-coverage
   echo "CICO: ran tests without coverage"
 }
 
+function check_postgres_healthiness(){
```

Comment:
ATM, This is an infinite loop that will be limited by the job timeout. However I'll add an additional retry-max-counter to avoid blocking other jobs.